labs/lab-angularjs/form/form.component.js

```diff
@@ -0,0 +1,11 @@
+angular.module("todoApp").component("todoform", {
+  templateUrl: "form/form.html",
+  controller: ['ItemService', function(ItemService) {
+    this.items = [];
+
+    this.items = function() {
```

Comment:
Here, you redefine it as a function. In this case the function is the right approach; I'd delete line 4.